oak_containers_orchestrator/src/key_provisioning.rs

```diff
@@ -13,29 +13,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::proto::oak::containers::{
-    key_provisioning_server::KeyProvisioning, GetProvisionSecretsRequest,
-    GetProvisionSecretsResponse, SendProvisionSecretsRequest,
+use crate::proto::oak::{
+    containers::{
+        orchestrator_key_provisioning_server::OrchestratorKeyProvisioning,
+        SendProvisionSecretsRequest,
+    },
+    key_provisioning::v1::{
+        key_provisioning_server::KeyProvisioning, GetProvisionSecretsRequest,
+        GetProvisionSecretsResponse,
+    },
 };
 use tonic::{Request, Response};
 
 struct KeyProvisioningService {}
 
+#[tonic::async_trait]
+impl OrchestratorKeyProvisioning for KeyProvisioningService {
+    async fn send_provision_secrets(
+        &self,
+        _request: Request<SendProvisionSecretsRequest>,
+    ) -> Result<Response<()>, tonic::Status> {
+        // TODO(#4442): Implement replacing group encryption key.
+        Ok(tonic::Response::new(()))
```

Comment:
Why does this return Ok if it's not implemented? Could it return an unimplemented error?